core/templates/dev/head/domain/utilities/AudioLanguageObjectFactorySpec.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for object factory for creating audio languages
+ *
+ */
+
+describe('AudioLanguage object factory', function() {
+  var audioLanguage;
```

Comment:
Initialize this (and below) with null.

Reply:
Thanks for noticing this. I was wondering whether I should initialize it as null too.